implementing Codespell into redo style build process

Jbsco opened the pull request:
Reopening #11

dinkelk commented on redo/rules/build_style.py:
```diff
@@ -85,6 +85,28 @@ def _build(self, redo_1, redo_2, redo_3):
+                          + os.path.basename(style_log_file)
+                          + " 2>&1 | tee -a "
+                          + style_log_file
+                          + " 1>&2")
```
Maybe try to combine some of the code duplication here to make maintenance easier.

Jbsco commented:
> I think this is not quite right. I think the directory you want to run this on is `directory` which is set near the top of the class. The current directory is usually just the repo root. What you want is the directory that the user ran `redo` from which is passed by the `redo_1`, etc. args.

Addressed in [a5fe26e](https://github.com/lasp/adamant/pull/12/commits/a5fe26e42536fd8e9e81fa279441414561c35980).

> Maybe try to combine some of the code duplication here to make maintenance easier.

Also addressed in [a5fe26e](https://github.com/lasp/adamant/pull/12/commits/a5fe26e42536fd8e9e81fa279441414561c35980).
